.github/workflows/main.yml

```diff
@@ -4,6 +4,12 @@ on:
   push:
   pull_request:
   workflow_call:
+    inputs:
+      tag_name:
+        description: 'The tag name'
+        default: ''
```

Comment:
The default is `''` and we're checking below `inputs.tag_name == false`. Is this working correctly? maybe we should make it `boolean` instead of `string` and default to `false`, or if we are using `string` we should compare with the empty string below.